client/config/cmd.go

```diff
@@ -44,11 +42,13 @@ func runConfigCmd(cmd *cobra.Command, args []string) error {
 
 	case 1:
 		// it's a get
-
 		key := args[0]
+
 		switch key {
 		case flags.FlagChainID:
 			cmd.Println(conf.ChainID)
+		case flags.FlagKeyringDir:
+			cmd.Println(conf.KeyringDir)
```

Comment:
Like I said, I don't think this should be a config value...

Reply:
Done